components/form/checkbox/README.md

```diff
@@ -0,0 +1,9 @@
+### FormCheckbox
+
+Simple checkbox form element.
+
+You can pass a custom callback to be executed when the checkbox value is changed, through ```onChange``` property.
```

Comment:
FYI you can use simple quotes instead of triple when it's not multiline

Reply:
Thanks! 👍
I will update this in next commit.